src/datadoc_editor/frontend/callbacks/variables.py

```diff
@@ -67,21 +64,10 @@ def populate_variables_workspace(
                     "Maskingenerert",
                     variable,
                 ),
-                *(
-                    [
-                        build_variables_pseudonymization_section(
-                            PSEUDONYMIZATION_METADATA,
-                            "Pseudonymisert",
-                            variable,
-                            variable.pseudonymization,
-                        )
-                    ]
-                    if variable.pseudonymization is not None
-                    else [
-                        build_variables_pseudo_button(
-                            "Pseudonymisert", short_name=variable.short_name or ""
-                        )
-                    ]
+                build_variables_pseudonymization_section(
+                    "Pseudonymisert",
```

Comment:
```suggestion
                    "Pseudonymisering",
```

I've always found this name a bit odd, since the metadata itself isn't pseudonymized. Up to you though, this is just a suggestion.